cmd/stellar-rpc/internal/ingest/service.go

```diff
@@ -295,12 +295,30 @@ func (s *Service) ingest(ctx context.Context, sequence uint32) error {
 		return err
 	}
 
-	// EvictedTemporaryLedgerKeys will include both temporary ledger keys which
-	// have been evicted and their corresponding ttl ledger entries
-	evictedTempLedgerKeys, err := ledgerCloseMeta.EvictedTemporaryLedgerKeys()
+	// EvictedTemporaryLedgerKeys will, in this completely borked up version of
```

Comment:
good catch, I forgot about this change. I think I'll need to rename `EvictedTemporaryLedgerKeys()` to `EvictedLedgerKeys()` in the go monorepo